.env.appStore.example

```diff
@@ -62,6 +62,11 @@ HUBSPOT_CLIENT_SECRET=""
 MS_GRAPH_CLIENT_ID=
 MS_GRAPH_CLIENT_SECRET=
 
+#   - SHIMMER
+# Used for the Shimmer Video App integration
+SHIMMER_API_KEY=
+SHIMMER_API_ROUTE=
+
```

Comment:
I think we can move them into the Apps Admin Wizard and save it in the DB.

ideally we dont want to add too many things to the .env

Reply:
Ok yeah that makes sense, what would be the best way to do this?